### PR TITLE
homepage-ui-shop-by-category added

### PR DESCRIPTION
**Pull request by ShakirHajabba**

### CODE REVIEW

File: src/backend/catergoryData.ts [hunk 1]
```diff
@@ -0,0 +1,99 @@
+export type categoryType = { key : number; category: string; url : string }[]
```
Comment (prateekoli1) [inline src/backend/catergoryData.ts:1]: remove this file, and send the mock data from home.ts file in backend.

File: src/components/shopByCategory/shopByCategory.component.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import { getCategoryData } from 'backend/user';
+import React, { useEffect, useState } from 'react';
+import styles from './shopByCategory.module.scss';
+
+console.log(styles);
```
Comment (prateekoli1) [inline src/components/shopByCategory/shopByCategory.component.tsx:5]: Remove console.log

File: src/components/shopByCategory/shopByCategory.component.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    const [categories, setCategories] = useState([]);
+
+    useEffect(()=>{
+        const resp = getCategoryData();
+        setCategories(resp);
+    },[categories]);
+
+  return (
+        <>
+            <div className={styles.stripContainer}>
```
Comment (prateekoli1) [inline src/components/shopByCategory/shopByCategory.component.tsx:19]: Fix indentation

File: src/components/shopByCategory/shopByCategory.component.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+                                </li>
+                            );
+                        })
+                    }
+                </ul>
+
+            </div>
+            
+            
+            </div>
```
Comment (prateekoli1) [inline src/components/shopByCategory/shopByCategory.component.tsx:49]: Remove empty lines

File: src/backend/catergoryData.ts (hunk 1 above)
Comment (ShakirHajabba) [reply src/backend/catergoryData.ts:1]: file removed. And mock data sent to home.ts

File: src/components/shopByCategory/shopByCategory.component.tsx (hunk 2 above)
Comment (ShakirHajabba) [reply src/components/shopByCategory/shopByCategory.component.tsx:5]: console.log removed.

File: src/components/shopByCategory/shopByCategory.component.tsx (hunk 3 above)
Comment (ShakirHajabba) [reply src/components/shopByCategory/shopByCategory.component.tsx:19]: Fixed.

File: src/components/shopByCategory/shopByCategory.component.tsx (hunk 4 above)
Comment (ShakirHajabba) [reply src/components/shopByCategory/shopByCategory.component.tsx:49]: Empty Lines removed.

**Comment by ShakirHajabba** [comment]
shopByCategory_1

**Comment by ShakirHajabba** [comment]
shopByCategory_FIXED

**Pull request merged**